### DISCUSSION TITLE
Can't re-verify the academic affiliation

### DISCUSSION THREAD

**Discussion by FAHRIZTX** [community/community#57757] [GitHub Education]
### Hi everyone,

Hi GitHub Education Team,

My Campus gives me an email with a `@surel.untag-sby.ac.id` domain for Student but GitHub only uses `@untag-sby.ac.id` domain for verification, can you help me to add `@surel.untag-sby.ac.id` domain to re-verify my GitHub student pack, because  I always get error message `We require applicants to use a school-issued email address to apply.`.

Thank You :)

### A suggestion

Adding `@surel.untag-sby.ac.id` domain to my university.
My University is: University August 17, 1945 (Universitas 17 Agustus 1945) (UNTAG), Surabaya

**Comment by mwt** [comment]
The easiest way is to get someone from your school's IT department to email GitHub education support.

**Comment by samus-aran** [comment]
Hi @FAHRIZTX 👋 

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
